config.yaml

```
@@ -6,6 +6,8 @@ parameters:
   shop:
     # 'helloasso', 'mangopay' and in future 'stripes', 'mollie' or other
     serviceName: 'helloasso'
+    lists:
+      ListeFormsProduit_id: 'ListeFormsProduit'
   shop_editable_config_params:
     - shop:
       - serviceName
```

```suggestion
      - serviceName
      - forms
        - products
```
Je rajouterais bien ce paramètre dans `shop_editable_config_params` pour pouvoir l'éditer depuis la page `EditConfig`. Il faudra alors définir dans les fichiers de langue les bonnes clés pour la traduction (exemple :  `EDIT_CONFIG_HINT_SHOP[FORMS][PRODUCTS]`)

En indiquant, de séparer les identifiants par des virgules et de faire `/update` pour leur création

---

@Eliotboutet j'ai dû mal m'exprimer. Il faudrait avoir le paramètre `forms` présent à la fois pour la clé `parameters[shop]` et pour la clé `parameters[shop_editable_config_params][shop]`